app/controllers/books_controller.rb

```diff
@@ -0,0 +1,11 @@
+class BooksController < ApplicationController
+  def index
+    @books = Book.all
```

Comment:
There is no need to use a instance variable, you can do `render_paginated Book.all`